stubs/pycurl/METADATA.toml

```diff
@@ -1,5 +1,14 @@
 version = "7.45.*"
 
 [tool.stubtest]
+ignore_missing_stub = false
+# Install on Windows requires building PycURL from source
+#
+# Install on MacOS is too complicated for the CI and does not work with stubtest:
+# % brew install openssl
+# % export LDFLAGS="-L/usr/local/opt/openssl@3/lib"
+# % export CPPFLAGS="-I/usr/local/opt/openssl@3/include"
+# % pip install --compile --install-option="--with-openssl" pycurl
 platforms = ["linux"]
 apt_dependencies = ["libcurl4-openssl-dev"]
+brew_dependencies = ["openssl"]
```

Comment:
Do we need this line if we don't run `macos` in CI?

Reply:
Likely not. Especially since atm the CI will install everything even if stubtest isn't run.

Reply:
Let's leave it out in that case (you can add a comment if you like) -- it'll only slow down the CI unnecessarily

Reply:
Yep, that's my plan. Now that it's proven the install works on CI, I've commented out unneeded installs and removed redundant platforms.